src/instructlab/process/process.py

```diff
@@ -427,3 +427,50 @@ def get_latest_process() -> str | None:
     last_key = list(process_registry.processes.keys())[-1]
     assert isinstance(last_key, str)
     return last_key
+
+
+def remove_process_record(input_uuid: str):
```

Comment:
is it going to remove processes that are still running? this will leave orphans that are not listed in `process list`. We should probably kill the process (meaning going through the regular `stop_process` flow). What do you think?